apps/server/src/modules/registration/api/dto/response/registration-list.response.ts

```diff
@@ -0,0 +1,11 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { RegistrationItemResponse } from './registration-item.response';
+
+export class RegistrationListResponse {
+	constructor({ data }: RegistrationListResponse) {
+		this.data = data;
+	}
+
+	@ApiProperty({ type: [RegistrationItemResponse] })
+	public data: RegistrationItemResponse[];
```

Comment:
We should probably have seperate response DTOs for the Listitem (in the room), and the registration object the invited user gets when he is starting the registration. That way we can control who can see what information.

Reply:
I agree, but as currently the response only would consist of:
- id
- email
- firstName
- lastName
- createdAt
- updatedAt

there would be (in my understanding) no difference in the response for the registering user and the response for the list in the room.
If we differentiate these two responses further I agree in adding another DTO, otherwise I would leave it as is for now.